tests/generic_config_updater/test_multiasic_addcluster.py

```diff
@@ -147,36 +156,45 @@ def test_addcluster_workflow(duthost):
         'PORTCHANNEL_MEMBER': set(),
         'DEVICE_NEIGHBOR': set()
     }
+
+    # Determine prefix for paths created by generate_config_patch.
+    # For multi-ASIC devices patches will be under '/<asicid>/...'; for single-ASIC they start with '/'.
+    def _prefix():
+        return f"/{ASICID}" if getattr(duthost, 'is_multi_asic', False) else ""
+
+    prefix = _prefix()
+
     for patch_entry in json_patch:
         path = patch_entry.get('path', '')
-        if path.startswith(f'/{ASICID}/PORT/'):
+        # Normalize checks to account for optional ASIC prefix
+        if path.startswith(f'{prefix}/PORT/') or path.startswith('/PORT/'):
```

Comment:
Can you help me understand why we need patch for PORT table? I think the port should already be there but admin down. What will be changed in the patch?